docs/composedb/set-up-your-environment.mdx

```diff
@@ -21,6 +21,31 @@ There are a few ways to set up your environment. Choose the one that best fits y
 - [Using the Wheel](#installation-using-wheel) - the recommended and the easiest way to configure your full working environment and install the necessary dependencies.
 - [Using JavaScript package managers](#installation-using-javascript-package-managers) - an alternative, more manual, way to configure your working environment which supports `npm`, `pnpm` and `yarn`.
 
+**Install and start the `ceramic-one` binary**
+
+All of the configuration options listed above **require a `ceramic-one` binary up and running**, which provides a data network access. You can run `ceramic-one` on your
+local machine using two simple steps:
+
+1. Install the component using [Homebrew](https://brew.sh/):
+
+```bash
+brew install ceramicnetwork/tap/ceramic-one
+```
+
+2. Start the `ceramic-one` using the following command:
+```bash
+ceramic-one daemon 
+```
+
+:::note
+By default, the command above will spin off a node which will connect to a `clay-testnet`. You can change this behaviour by providing a `--network` flag and specifying a network of your choice. For example:
```

Comment:
By default ceramic-one will store its data in the current directory. We should provide an example of starting ceramic-one with args to place the data in a specific directory.

```
ceramic-one daemon --store-dir ~/.ceramic-one --p2p-key-dir ~/.ceramic-one
```